manifests/install/rabbitmqadmin.pp

```diff
@@ -3,6 +3,14 @@
 
   require '::archive'
 
+  # Newer versions of Ubuntu (16.04) ship with Python 3 only by default.
+  if $facts['os']['family'] == 'Debian' {
+    ensure_packages('python')
+    $rabbitmqadmin_require = [Archive['rabbitmqadmin'], Package['python']]
```

Comment:
Not sure if there's a cleaner way to do this

Reply:
also, this logic needs to be moved to params, and I think it probably does need to be configurable.